src/backend_stub/index.js

```diff
@@ -18,6 +18,13 @@ module.exports = {
 		var app = express(http.createServer());
 		var appServer;
 		
+		app.use(config.staticBaseUrl, serveStatic(config.staticPath));
+		app.get('/', function (req, res) {
+			res.sendFile('index.html', {
```

Comment:
`backend_stub` won't be used in production. It is entirely replaced with C++ backend. It's for testing Web independently.